01-main/packages/edrawmind

```diff
@@ -0,0 +1,12 @@
+DEFVER=1
+get_website "https://www.edrawsoft.com/download-edrawmind.html"
+if [ "${ACTION}" != prettylist ]; then
+    URL="$(grep -E -o -m 1 "https://[^\"]*\.deb" "${CACHE_FILE}")"
+    VERSION_PUBLISHED="$(grep -m 1 -E -i -A 10 --no-group-separator "for linux" "${CACHE_FILE}" | grep -m 1 -E -o "[0-9]+\.[0-9]+\.[0-9]+")"
```

Comment:
The page no longer seems to present the version of the current release so this fails.